src/lib/DailyTodosCard/DailyTodo.svelte

```diff
@@ -26,5 +26,6 @@
 
     label {
         display: flex;
+        align-items: center;
```

Comment:
I'm always on firefox so i was always to lazy to fix this alignment issue, thank you!